src/Sulu/Bundle/SecurityBundle/UserManager/UserManager.php

```diff
@@ -476,7 +468,7 @@ private function addUserRole(UserInterface $user, $userRoleData)
     /**
      * Returns the contact with the given id.
      *
-     * @param int $id
+     * @param int|string|null $id
```

Comment:
This should be kept as it is, unless what the callers currently does in core. To project outside it should only be called officially with int.

```suggestion
     * @param int $id
```